doc/sphinx/source/introduction.rst

```diff
@@ -50,12 +50,13 @@ these challenging objectives the ESMValTool is conceived as a framework
 which allows community contributions to be bound into a coherent
 framework.
 
-.. _contact:
+.. _Support-and-Contact:
 
-Contact
+Support
 =======
 
-See `www.esmvaltool.org <https://www.esmvaltool.org>`_ for general contact information.
+Support for ESMValTool can be found here `ESMValTool Discussions page <https://github.com/ESMValGroup/ESMValTool/discussions>`__
+where users can open an issue and a member of the User Engangement Team of ESMValTool will reach you.
 
```

Comment:
```suggestion
where users can open an issue and a member of the User Engagement Team of ESMValTool will reach you. The user engagement team can be contacted at [esmvaltool_user_engagement_team@listserv.dfn.de](mailto:esmvaltool_user_engagement_team@listserv.dfn.de)
```
I think it would be good to add the UET email address here. And also to move the ESMValTool mailing-list to the next section "Contact". If the 2 email addresses are displayed together, I think some users would be confused which one to use. Also the ESMValTool mailing-list is more of a diffusion list (monthly meetings annoucements...) so it would make sense to have the information displayed right before/after the Monthly Meetings.